test/unit/communication/test_async.cpp

```diff
@@ -0,0 +1,43 @@
+//
+// Distributed Linear Algebra with Future (DLAF)
+//
+// Copyright (c) 2018-2019, ETH Zurich
+// All rights reserved.
+//
+// Please, refer to the LICENSE file in the root directory.
+// SPDX-License-Identifier: BSD-3-Clause
+//
+
+#include <bits/c++config.h>
```

Comment:
Out of curiosity: what is this for?

Reply:
`clangd` occasionally adds some header it thinks are needed, I haven't noticed that one, I will remove it.